cmd/microcloud/preseed.go

```diff
@@ -19,14 +22,16 @@ import (
 
 	"github.com/canonical/microcloud/microcloud/api"
 	"github.com/canonical/microcloud/microcloud/api/types"
+	cloudClient "github.com/canonical/microcloud/microcloud/client"
 	"github.com/canonical/microcloud/microcloud/mdns"
 	"github.com/canonical/microcloud/microcloud/service"
 )
 
 // Preseed represents the structure of the supported preseed yaml.
 type Preseed struct {
-	LookupSubnet          string        `yaml:"lookup_subnet"`
```

Comment:
I thought we still needed a LookupSubnet for the multicast case where we don't know the joiners' addresses precisely. 

Can't we set a subnet and pick the address of the first interface that fits in the subnet as the initiator's listen address, and then use mDNS to pick up the remaining systems, filtering by this subnet?

multicast should really be a best effort so if it's not precise, that's fine I think. If you need precision you'd use unicast.

Reply:
Please see the comment here https://github.com/canonical/microcloud/pull/383#discussion_r1736694656.